spec/pallenec_spec.lua

```diff
@@ -46,6 +46,12 @@ describe("pallenec", function()
     it("Can detect conflicting arguments", function()
         local ok, err, _, abortMsg = util.outputs_of_execute("./pallenec --emit-c --emit-asm __test__.pln")
         assert.is_false(ok, err)
-        assert.equals("./pallenec: flags --emit-c and --emit-asm are mutually exclusive\n", abortMsg)
+        assert.equals([[Usage: pallenec
```

Comment:
This assert.equals will break whenever we change the set of flags. What do you think about doing a substring search instead?

    assert(string.find(err, "Error: option '--emit-asm' can not be used together with option '--emit-c'", nil, true))